march_hardware/include/march_hardware/encoder/Encoder.h

```diff
@@ -36,6 +36,13 @@ class Encoder
    */
   virtual double toRad(int32_t iu) const = 0;
 
+  /*
```

Comment:
Necessary for docstrings
```suggestion
  /**
```